src/main/java/de/unistuttgart/overworldbackend/data/NPCTask.java

```diff
@@ -0,0 +1,26 @@
+package de.unistuttgart.overworldbackend.data;
```

Comment:
Why is it named `NPCTask`? In our Database Model it is named `NPC`

Reply:
changed to NPC